ideasbox/templates/index.html

```diff
@@ -3,17 +3,18 @@
 {% load i18n ideasbox_tags %}
 
 {% block content %}
-    <div class="row">
-        <div class="col half">
-            <h2 class="flow"><span class="theme create">blog</span><span>{% trans "LAST POSTS" %}</span></h2>
-            {% for content in blog_contents %}
-                {% include 'blog/content_card.html' %}
-            {% endfor %}
-        </div>
-        <div class="col half">
-            {% include 'search/box.html' %}
+    <div class="row column_small_screen">
```

Comment:
We have change the ".column-smallscreen" into "mwide" or "twide" but we can't push it now, i'll do it tmw morning